src/__stories__/menu-story.tsx

```diff
@@ -22,7 +28,11 @@ const options = [
     },
 ];
 
-export const Default: StoryComponent = () => {
+type MenuArgs = {
+    align: 'right' | 'left';
```

Comment:
to me `align` is a bad name (you aren't aligning), I'd use `openDirection` or just `direction`

Reply:
changed, but.... now that is called direction, `right` should be the default value? as the menu that open from the target to the right (as it is in the attached screenshot at the description of the PR) will be called right?

Reply:
in Toltip we call this `position`

Reply:
changed